server/routerlicious/packages/routerlicious-base/src/utils/sessionHelper.ts

```diff
@@ -129,6 +137,10 @@ async function updateExistingSession(
                 session: updatedSession,
             },
             null);
+        Lumberjack.info(
+            `The Session ${JSON.stringify(updatedSession)} was updated into the document collection`,
```

Comment:
Instead of logging the session information in the message, do we instead want to log it through the `properties` object? Basically, a similar idea to what was mentioned in this comment: https://github.com/microsoft/FluidFramework/pull/12574/commits/25bb747433a24db3742d5fee9a91e9c159de6faf#r999732450

Reply:
Let's keep it to be consistent with line 45. Also, the updated session is not very long to print out, so we can put it into the message, and the user can directly see it.